python/lsst/ts/mtdometrajectory/dome_trajectory.py

```diff
@@ -501,11 +515,18 @@ async def follow_target(self):
                 telescope_target=self.telescope_target,
                 next_telescope_target=self.next_telescope_target,
             )
-            if desired_dome_azimuth is not None:
+            if desired_dome_azimuth is not None and not (
+                math.isnan(desired_dome_azimuth.position)
+                or math.isnan(desired_dome_azimuth.velocity)
+            ):
                 moved_azimuth = True
                 self.move_dome_azimuth_task = asyncio.create_task(
                     self.move_dome_azimuth(desired_dome_azimuth)
                 )
+            else:
+                self.log.warning(
```

Comment:
Consider rephrasing:

```
f"{desired_dome_elevation=} too small or invalid; not moving dome elevation."
```

However, now that in think of it, why it being "too small" would be a problem?

Reply:
I had the same doubt here. The configuration plus this part of the code prevents the dome from moving if the difference in azimuth, corrected for altitude, is smaller than 5 degrees which I think is much too large. something we need to discuss at some point. I'll ping you on Slack about it.